setup.py

```diff
@@ -112,6 +99,8 @@ def build_extension(self, ext):
         )
 
 
+# The information here can also be placed in setup.cfg - better separation of
+# logic and declaration, and simpler if you include description/version in a file.
```

Comment:
Any reason to give this advice but not apply it ourselves? I don't mind either way, but I could see a confused user saying "well, why don't you?".

Reply:
I'd be fine with that, I'm a big fan of putting all declarations in setup.cfg, and leaving setup.py for only compile logic. The reason it's here in this example is that it keeps the example in one place, and because it's just an example, the parts where it makes a big difference (reading README files, adding a Version, etc) are not included in the example.

Reply:
Right. As you prefer. I just noticed there's a bit of a "do as I say, not as a I do" thing going on. But I agree it makes sense in the context of a minimal example.